python/mqt/syrec/simulation_view/dialogs/base_progress_dialog.py

```
@@ -0,0 +1,315 @@
+
+    Inputs:
+        WorkerType: Defines the type of worker employed by the dialog to perform its long running operation.
+    """
+
+    def __init__(
+        self,
+        parent: QtWidgets.QWidget,
+        shared_simulation_runs_model: QtSimulationRunModel,
+        dialog_title: str,
+        optional_progress_bar_text_format: str | None = None,
+        create_default_layout: bool = True,
```

Ruff (0.14.14)

[error] Boolean-typed positional argument in function definition

(FBT001)